src/AltinnCore/Common/Services/Implementation/GiteaAPIWrapper.cs

```diff
@@ -245,5 +245,58 @@ public string CreateAppToken(string name)
                 return null;
             }
         }
+
+        /// <summary>
+        /// Gets a list over the organizations that the current user has access to.
+        /// </summary>
+        /// <param name="giteaSession">The sessionId</param>
+        /// <returns>A list over all</returns>
+        public async Task<List<AltinnCore.RepositoryClient.Model.Organization>> GetUserOrganizations(string giteaSession)
+        {
+            List<AltinnCore.RepositoryClient.Model.Organization> organizations = null;
+            DataContractJsonSerializer serializer = new DataContractJsonSerializer(typeof(List<AltinnCore.RepositoryClient.Model.Organization>));
+            Uri giteaUrl = null;
+            Cookie cookie = null;
+
+            // TODO: Figure out how appsettings.json parses values and merges with environment variables and use these here
+            // Since ":" is not valid in environment variables names in kubernetes, we can't use current docker-compose environment variables
```

Comment:
@jezpoz It seems you can use double-underscore (__) instead of colon to get that working in docker
https://stackoverflow.com/questions/48298284/merging-appsettings-with-environment-variables-in-net-core

Do we have a task/issue on this or should we create one?@jezpoz